src/App.css

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Lilita+One&display=swap");
```

Comment:
Can leave as is. You could also add google fonts to index.html like we usually do.